src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -776,9 +776,25 @@ The accounting information can be accessed only through a *JMX client* or the Pr
 [[_run_as_me]]
 == Run Computation with a user's system account
 
-When starting a ProActive Node, it can be configured to run tasks under the user's system account when
-link:ProActiveUserGuide.html#_run_computation_with_your_system_account[runAsMe] is set to true in
-the job's definition. The default is to use the password method.
+Configure a ProActive Node to execute tasks under a user's system account by ticking the
+link:ProActiveUserGuide.html#_run_computation_with_your_system_account[Run as me] box in the task configuration.
+By default authentication is done through a password, but can also be configured using a SSH key.
+
+
+For proper execution, *the user's system account must have:*
+
+* Execution rights to the PROACTIVE_HOME directory and all it's parent directories.
+* Write access to the PROACTIVE_HOME directory.
+
+Logs are written during task execution, as a matter of fact, the executing user needs write access to the
+PROACTIVE_HOME directory. ProActive does some tests before executing as a different user, those need full execution
+rights, including the PROACTIVE_HOME parent directories.
+
+TIP: Create a ProActive group which owns the PROACTIVE_HOME directory and has write access. Add every system user, which
+executes tasks under it's own system account, to the ProActive group.
+
+Find a a step by step tutorial link:AdminTutorials.html[here].
```

Comment:
`gradle clean build`, then when I open build/asciidoc/html5/index.html in
my browser, usually links work.

On Mon, Nov 30, 2015 at 4:56 PM tobwiens <notifications@github.com> wrote:

> In src/docs/admin/ProActiveAdminGuide.adoc
> <https://github.com/ow2-proactive/documentation/pull/244#discussion_r46161043>
> :
>
> > +By default authentication is done through a password, but can also be configured using a SSH key.
> > +
> > +
> > +For proper execution, *the user's system account must have:*
> > +
> > +* Execution rights to the PROACTIVE_HOME directory and all it's parent directories.
> > +* Write access to the PROACTIVE_HOME directory.
> > +
> > +Logs are written during task execution, as a matter of fact, the executing user needs write access to the
> > +PROACTIVE_HOME directory. ProActive does some tests before executing as a different user, those need full execution
> > +rights, including the PROACTIVE_HOME parent directories.
> > +
> > +TIP: Create a ProActive group which owns the PROACTIVE_HOME directory and has write access. Add every system user, which
> > +executes tasks under it's own system account, to the ProActive group.
> > +
> > +Find a a step by step tutorial link:AdminTutorials.html[here].
>
> Thank you, corrected.
> How did you test the link? For me, all links in the documentation are
> broken, except the ones referring to the same document.
> Doesn't the deployment script copy the html files and then, the links work?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ow2-proactive/documentation/pull/244/files#r46161043>.
>